lib/blocktree/blocktree.go

```diff
@@ -68,6 +70,50 @@ func (bt *BlockTree) GenesisHash() Hash {
 	return bt.head.hash
 }
 
+// GetAllHashesForParentDepth will get all hashes with the same dept for a given header
+func (bt *BlockTree) GetAllHashesForParentDepth(header *types.Header) (map[common.Hash]*big.Int, error) {
```

Comment:
the name of this is a confusing, you should be getting all the hashes at the same depth as the header you pass in

Reply:
outdated

Reply:
this name of this doesn't correctly explain what this function should do, also it doesn't need to return a map, just an array of hashes